public/v4/apps/weblate.yml

```diff
@@ -0,0 +1,88 @@
+        volumes:
+            - $$cap_appname-data:/app/data
+        restart: always
+        environment:
+            DEBUG: 0
+            WEBLATE_LOGLEVEL: INFO
+            WEBLATE_ADMIN_EMAIL: $$cap_admin_email
+            WEBLATE_ADMIN_PASSWORD: $$cap_admin_password
+            WEBLATE_SERVER_EMAIL: $$cap_server_email
+            WEBLATE_DEFAULT_FROM_EMAIL: $$cap_email_from
+            WEBLATE_REGISTRATION_OPEN: $$cap_open_registration
+            POSTGRES_PASSWORD: $$cap_pg_pass
+            POSTGRES_USER: weblate
+            POSTGRES_DATABASE: weblate
+            POSTGRES_HOST: srv-captain--$$cap_appname-postgres
+            REDIS_HOST: srv-captain--$$cap_appname-redis
+            WEBLATE_SITE_DOMAIN: $$cap_appname.$$cap_root_domain
+            WEBLATE_SITE_TITLE: $$cap_appname Weblate
+            WEBLATE_ADMIN_NAME: Weblate Admin
+            POSTGRES_PORT: 5432
+            REDIS_PORT: 6379
+        caproverExtra:
+            containerHttpPort: '8080'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_postgres_version
+          label: Postgres Docker version
+          defaultValue: 13-alpine
+          description: Check out their Docker page for the valid versions https://hub.docker.com/r/library/postgres
+        - id: $$cap_redis_version
+          label: Redis Docker version
+          defaultValue: 6-alpine
+          description: Check out their Docker page for the valid versions https://hub.docker.com/r/library/redis
+        - id: $$cap_weblate_version
+          label: weblate Docker Image SHA Hash
+          defaultValue: latest
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/weblate/weblate
+        - id: $$cap_pg_pass
+          label: Postgres Password
+          description: Password must be at least 30 characters.  Please use a random string.
+          validRegex: /^[^\@]{30,}$/
+        - id: $$cap_admin_email
+          label: Email of the first user and administrator of weblate
+          description: Will be used to login to weblate
+        - id: $$cap_admin_password
+          label: Password of the first user and administrator of weblate
+          description: Will be used to login to weblate
+        - id: $$cap_server_email
+          label: Weblate server email
+          description: (eg weblate@example.com)
+        - id: $$cap_email_from
+          label: Weblate FROM email
+          description: (eg weblate@example.com)
+        - id: $$cap_open_registration
+          label: Allow open registration of people (0/1)
+          defaultValue: 0
+          description: Do you allow people to register openly to the platform?
+    instructions:
+        start: Open-source collaborative localization tools
+        end: weblate is deployed and available as srv-captain--$$cap_appname:80.  Note that the application may take up to ten minutes to become available.
+    displayName: weblate
+    isOfficial: false
```

Comment:
weblate/weblate is official image so you can set
isOfficial: false to true

Reply:
Oh! I thought official meant that this was something that this was some sort of official apps maintained by the CapRover project (as in, not the community). Thanks I'll fix this :)